app/models/job_execution.rb

```diff
@@ -202,6 +202,7 @@ def commands(dir)
       PROJECT_REPOSITORY: @job.project.repository_url,
       REFERENCE: @reference,
       REVISION: @job.commit,
+      COMMIT_RANGE: @job.deploy.changeset.commit_range,
```

Comment:
not every job has a deploy ... so only set it if it has a deploy

Reply:
also double-check that it's a string ...

Reply:
Updated, let me know what you think. Apologies, I'm an infra engineer with not much dev background - learning as I go along!